tools/jsoncons/include/jsoncons/detail/unicode_traits.hpp

```diff
@@ -261,10 +261,12 @@ is_legal_utf8(Iterator first, size_t length)
         if (((a = (*--srcptr))& 0xC0) != 0x80)
             return conv_errc::expected_continuation_byte;
         // FALLTHRU
+    LLVM_FALLTHROUGH;
```

Comment:
Change these to the ```[[fallthrough]]``` attributes.  This will fail to compile with GCC.

Reply:
Ah I remember why I didn't do that, it throws a warning due to `-Wc++17-extension`

Reply:
Shoot okay.  Then use the goofy GCC/Clang kludge `/* fall through */`.

Reply:
Hmm I've tried several different variants of that comment and am not having any luck. Does that definitely work in clang or only in gcc?